Core/Importers/PrivatTransactionsImporter.cs

```diff
@@ -8,10 +7,11 @@
 
 namespace Core.Importers
 {
-    public class PrivatTransactionsImporter : TransactionsImporter
+    public class PrivatTransactionsImporter : ITransactionsImporter
     {
-        public override List<Transaction> Load(Stream file)
+        public List<Transaction> Load(Stream file)
```

Comment:
Codacy found an issue: [Refactor this method to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)